acceptance/suites/pre_suite/upgrade/10_install_puppet.rb

```diff
@@ -1,5 +1,5 @@
-step "Install PC1 repository" do
-  install_puppetlabs_release_repo(hosts, 'pc1')
+step "Install puppet repository" do
+  install_puppetlabs_release_repo(hosts, 'puppet')
```

Comment:
Yes, that seems like a reasonable path forward. I'm talking to @kevpl at the moment, and it looks like any supporting methods around installing the release packages have been removed. So that alone will take some thought if you are going to upgrade beaker